spec/models/async_transaction/vet360/base_spec.rb

```diff
@@ -52,7 +52,7 @@
     end
 
     it 'persists the messages from vet360' do
-      VCR.use_cassette('vet360/contact_information/email_transaction_status_w_message') do
```

Comment:
I don't think this cassette is used anywhere now.  Can we delete it?

Reply:
sounds good, i deleted it